src/books/Tests/Books.BusinessTests/Handlers/BookInstancesCommandsHandlersTests.cs

```diff
@@ -0,0 +1,59 @@
+﻿using Xunit;
+using Moq;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Books.Infrastructure.Contracts;
+using Books.Infrastructure.Models;
+using Books.Business.Commands;
+using Books.Business.Handlers;
+
+namespace Books.BusinessTests.Handlers
+{
+    public class BookInstancesCommandsHandlersTests
+    {
+        private readonly Mock<IBookRepository<BookRead>> _bookReadRepositoryMock;
+        private readonly Mock<IBookInstancesRepository<BookInstance>> _bookInstanceRepositoryMock;
+
+        public BookInstancesCommandsHandlersTests()
+        {
+            _bookReadRepositoryMock = new Mock<IBookRepository<BookRead>>();
+            _bookInstanceRepositoryMock = new Mock<IBookInstancesRepository<BookInstance>>();
+        }
+
+        [Fact]
+        public async void CreateBookInstanceHandlerTest()
```

Comment:
Work on test names, use some kind of convention, e.g. one of presented in this [article](https://medium.com/@stefanovskyi/unit-test-naming-conventions-dd9208eadbea). Fix for all other occurrences as well.